backend/controllers/alert.js

```diff
@@ -11,14 +11,36 @@ const alertFactory = (oauthApi, elite2Api) => {
     const { alert, pageErrors, offenderDetails, ...rest } = pageData
     const formAction = offenderDetails && alert && `/api/close-alert/${offenderDetails.bookingId}/${alert.alertId}`
 
+    const {
+      active,
+      alertCode,
+      alertCodeDescription,
+      alertId,
+      alertType,
+      alertTypeDescription,
+      comment,
+      dateCreated,
+      expired,
+      addedByFirstName,
+      addedByLastName,
```

Comment:
Would it be better to do `const { addedByFirstName, addedByLastName, dateCreated, ...rest = alert || {{}`